package.json

```diff
@@ -17,9 +17,9 @@
   "dependencies": {
     "@walletconnect/chat-client": "^0.7.4",
     "@walletconnect/core": "^2.9.2",
-    "@walletconnect/notify-client": "^0.1.3",
+    "@walletconnect/notify-client": "^0.1.5",
     "@walletconnect/notify-message-decrypter": "^0.1.0",
-    "@walletconnect/sync-client": "^0.3.7",
+    "@walletconnect/push-message-decrypter": "^0.10.6",
```

Comment:
Can remove this since this pkg is stale now/duplicating its `notify-` equivalent above

```suggestion

```